apps/podium/app/comment/comment.tsx

```diff
@@ -0,0 +1,101 @@
+import styles from './comment.module.scss';
+import styled from 'styled-components';
+
+/* eslint-disable-next-line */
+export interface CommentProps {}
+
+export function Comment(props: CommentProps) {
+  return (
+    <OuterContainer>
+      <RowOne>
+        <Circle
+          src={
+            'https://st3.depositphotos.com/9998432/13335/v/450/depositphotos_133352062-stock-illustration-default-placeholder-profile-icon.jpg'
+          }
+          alt={'profile pic'}
+        />
+        <NameandReply>
+          <Name>Leyland Yang</Name>
+          <Reply>reply</Reply>
+        </NameandReply>
+      </RowOne>
+      <RowTwo>
+        <Body>
+          This is a comment from the judge, and everyone can see the comment
+        </Body>
+      </RowTwo>
+      <RowThree>
+        <TimeStamp>3 mins ago</TimeStamp>
+      </RowThree>
+    </OuterContainer>
+  );
+}
+const NameandReply = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  padding: 0 20px;
+`;
+const OuterContainer = styled.div`
+  width: 355px;
+  background: #161616;
```

Comment:
nit: Extract this (and all other colors) out into a const (ideally shared between components) so that we don't have random hex codes littered around the css